http/http-advanced-reactive/src/main/java/io/quarkus/ts/http/advanced/reactive/FootballTeamResource.java

```diff
@@ -0,0 +1,80 @@
+package io.quarkus.ts.http.advanced.reactive;
+
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.LinkedHashSet;
+import java.util.Map;
+import java.util.Set;
+
+import jakarta.ws.rs.Consumes;
+import jakarta.ws.rs.DELETE;
+import jakarta.ws.rs.POST;
+import jakarta.ws.rs.Path;
+import jakarta.ws.rs.Produces;
+import jakarta.ws.rs.core.MediaType;
+import jakarta.ws.rs.core.Response;
+
+import org.jboss.logging.Logger;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.JsonMappingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import io.smallrye.mutiny.Uni;
+
+/**
+ *
+ * When a JSON extension is installed such as quarkus-rest-jackson or quarkus-rest-jsonb
+ * Quarkus will use the application/json media type by default for most return values
+ */
+@Path("/football")
+public class FootballTeamResource {
+
+    private static final Logger LOG = Logger.getLogger(FootballTeamResource.class);
+
+    private Set<FootballTeam> footballTeamSet = Collections.synchronizedSet(new LinkedHashSet<>());
```

Comment:
flow is:

1. `String data` is deserialized with `Set<FootballTeam> teamSet = objectMapper.readValue(data, new TypeReference<Set<FootballTeam>>() {});`
2. you have `Set<FootballTeam> teamSet`
3. then you do validation (that part is ok)
4. then you copy it into a class member synchronized set
5. then you use the synchronized set to return the response
6. then you have separate call that deletes a football team set

Things that I don't understand:

- it's synchronized but there is no concurrency, you never call it more than once at time
- you copy `teamSet` into `footballTeamSet` that is kept outside of `/upload-football-json`, but you never use it; instead you only use it to "delete it"; what is the point of copying it into the `footballTeamSet`
- you need to manage `footballTeamSet` but it's not useful for you